ocaml-lsp-server/test/e2e-new/code_actions.ml

```diff
@@ -153,3 +159,152 @@ let f x = (1 : int :> int)
   in
   print_code_actions source range ~filter:find_annotate_action;
   [%expect {| No code actions |}]
+
+let%expect_test "can remove type annotation from a function argument" =
+  let source =
+    {ocaml|
+type t = Foo of int | Bar of bool
+let f (x : t) = Foo x
+|ocaml}
+  in
+  let range =
+    let start = Position.create ~line:2 ~character:7 in
+    let end_ = Position.create ~line:2 ~character:8 in
+    Range.create ~start ~end_
+  in
+  print_code_actions source range ~filter:find_remove_annotation_action;
```

Comment:
An idea that's unrelated to this PR:

It would be nice to actually apply the code actions and print the contents of the document. It would be much easier to verify the actions are correct this way.

Reply:
Yes, I was thinking about this too while writing these tests. Maybe we should create an issue describing this idea?

Reply:
Sounds good.